updater/fetchers/node/node.go

```diff
@@ -0,0 +1,212 @@
+// Copyright 2016 clair authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package node
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/coreos/clair/database"
+	"github.com/coreos/clair/updater"
+	cerrors "github.com/coreos/clair/utils/errors"
+	"github.com/coreos/clair/utils/types"
+	"github.com/coreos/pkg/capnslog"
+)
+
+const (
+	url               = "https://api.nodesecurity.io/advisories"
+	cveURLPrefix      = "http://cve.mitre.org/cgi-bin/cvename.cgi?name="
+	updaterFlag       = "nodeUpdater"
+	defaultNPMVersion = "all"
+	//FIXME: When an advisory is fixed after a certain version,
+	//  current Clair could not differentiate it, so add an suffix.
+	defaultVersionSuffix = "-1"
+)
+
+var log = capnslog.NewPackageLogger("github.com/coreos/clair", "updater/fetchers/node")
+
+type nodeCVE struct {
+	ID string
+}
+type nodeAdvisory struct {
+	ID                  int
+	Created_at          string
```

Comment:
These should be CamelCase and then use struct tags for their representation in JSON.